lib/ansible/plugins/lookup/conjur_variable.py

```diff
@@ -0,0 +1,146 @@
+        display.vvvv("configuration: {}".format(identity))
+
+        # Use credentials to retrieve temporary authorization token
+        conjur_url = "{}/authn/{}/{}/authenticate".format(conf['appliance_url'], conf['account'], identity['id'])
+        display.vvvv("Authentication request to Conjur at: {}, with user: {}".format(conjur_url, identity['id']))
+
+        if 'cert_file' in conf:
+            response = requests.post(conjur_url, data=identity['api_key'], verify=conf['cert_file'])
+        else:
+            response = requests.post(conjur_url, data=identity['api_key'])
+        display.vvvv("response: {}".format(response.text))
+
+        if response.status_code != 200:
+            raise AnsibleError('Failed to authenticate as \'{}\''.format(identity['id']))
+
+        # Retrieve Conjur variable using the temporary token
+        token = b64encode(response.text)
+        headers = {'Authorization': "Token token=\"{}\"".format(token)}
+        display.vvvv("Header: {}".format(headers))
+
+        url = "{}/secrets/{}/variable/{}".format(conf['appliance_url'], conf['account'], quote_plus(terms[0]))
+        display.vvvv("Conjur Variable URL: {}".format(url))
+
+        if 'cert_file' in conf:
+            response = requests.get(url, headers=headers, verify=conf['cert_file'])
+        else:
+            response = requests.get(url, headers=headers)
+
+        if response.status_code == 200:
+            display.vvvv("Conjur variable {} was successfully retrieved".format(terms[0]))
+            return [response.text]
+        if response.status_code == 401:
+            raise AnsibleError('Conjur request has invalid authorization credentials')
+        if response.status_code == 403:
+            raise AnsibleError('Conjur host does not have authorization to retrieve {}'.format(terms[0]))
+        if response.status_code == 404:
+            raise AnsibleError('The variable {} does not exist'.format(terms[0]))
+
+    # Load configuration and return as dictionary if file is present on file system
+    def load_conf(self, conf_path):
+        conf_path = os.path.expanduser(conf_path)
+
+        if conf_path and os.path.exists(conf_path):
+            display.vvvv("Loading configuration from: %s" % conf_path)
+            with open(conf_path) as f:
+                try:
+                    return yaml.safe_load(f.read())
+                except Exception as exception:
+                    display.info("Error: parsing %s - %s" % (conf_path, str(exception)))
+                    AnsibleError("Error: parsing %s - %s" % (conf_path, str(exception)))
+        return {}
+
+    # Load identity and return as dictionary if file is present on file system
+    def load_identity(self, identity_path, appliance_url):
+        identity_path = os.path.expanduser(identity_path)
+
+        if identity_path and os.path.exists(identity_path):
+            display.vvvv("Loading identity from: %s" % identity_path)
+            try:
```

Comment:
`try` block should contain fewer statements. `Exception` should be replaced with a more precise exception type.